### ISSUE TITLE
add_object() from user defined type does not inherit subnodes.

### ISSUE THREAD

**Issue by MikelAyani** [FreeOpcUa/python-opcua#573]
Hi,
I have seen after the last update today (using pip3) that when instancing (add_object()) user made ObjectType and VariableType, their children nodes are not inherited. I guess it is a bug in the last version because it worked before....
Thanks!

**Comment by oroulet** [comment]
Can you dobbel check that it is not a user error. I used it yesterday from master and it worked fine...

**Comment by MikelAyani** [comment]
Sorry, it works with ObjectTypes but not with VariableTypes which have Variable children.

**Comment by MikelAyani** [comment]
Sorry, it works with ObjectTypes but not with VariableTypes which have Variable children.

**Comment by oroulet** [comment]
You should open a bug report for that. We just allowed it and we should also instantiste children of variable... Even better make a pr

**Comment by MikelAyani** [comment]
Ok, thanks! But how can I open a bug report?

**Comment by cirp-usf** [comment]
I've encountered this problem before. As a work around I've been using add_object() to add variable types. I just pass a VariableType to add_object() and the children get instantiated.

**Comment by MikelAyani** [comment]
Yes, this is what I have been doing, but since last update add_object with a VariableType does not inherit children...
